modules/storage-aws-s3/s3/s3.go

```diff
@@ -93,7 +102,7 @@ func (s *s3) StoreSnapshot(ctx context.Context, snapshot snapshots.Snapshot) err
 	if err != nil {
 		return errors.Wrapf(err, "save meta")
 	}
-	objectName := fmt.Sprintf("%s/snapshot.json", snapshotID)
+	objectName := s.makeObjectName(snapshotID, "snapshot.json")
```

Comment:
snapshot is now also written to disk (see snapshot's `WriteToDisk` method). i guess it would be better to copy the existing one instead of creating it again

Reply:
Looks like there's a conflicting naming convention, one piece of code is using "snapshot.json", the other is "snapshot_id.json".  Which one is better?